EazyE2E.Console/App.config

```diff
@@ -3,4 +3,8 @@
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
     </startup>
+
```

Comment:
Config setting in the consuming application, this would presumably be the unit test project when we set that up.